src/extensions/sb-ncbr/partial-charges/labels.ts

```diff
@@ -0,0 +1,39 @@
+import { StructureElement, StructureProperties } from '../../../mol-model/structure';
+import { LociLabel } from '../../../mol-plugin-state/manager/loci-label';
+import { SbNcbrPartialChargesPropertyProvider } from './property';
+import { Loci } from '../../../mol-model/loci';
+import { PluginContext } from '../../../mol-plugin/context';
+import { LociLabelProvider } from '../../../mol-plugin-state/manager/loci-label';
+
+export function SbNcbrPartialChargesLociLabelProvider(ctx: PluginContext): LociLabelProvider {
+    return {
+        label: (loci: Loci) => {
+            if (!StructureElement.Loci.is(loci)) return;
+
+            const loc = StructureElement.Loci.getFirstLocation(loci);
+            if (!loc) return;
+
+            const granularity = ctx.managers.interactivity.props.granularity;
+            if (granularity !== 'element' && granularity !== 'residue') {
+                return;
+            }
+
+            const atomId = StructureProperties.atom.id(loc);
+            const model = loci.structure.model;
+            const data = SbNcbrPartialChargesPropertyProvider.get(model).value?.data;
+            if (data === undefined) return;
+            const { typeIdToAtomIdToCharge, typeIdToResidueToCharge } = data;
+
+            const typeId = SbNcbrPartialChargesPropertyProvider.getParams(model).typeId.defaultValue;
+            const showResidueCharge = granularity === 'residue';
+            const charge = showResidueCharge
+                ? typeIdToResidueToCharge.get(typeId)?.get(atomId)
+                : typeIdToAtomIdToCharge.get(typeId)?.get(atomId);
+            const label = granularity === 'residue' ? 'Residue charge' : 'Atom charge';
+
+            return `<strong>${label}: ${charge?.toFixed(4) || 'undefined'}</strong>`;
+        },
+        group: (label: LociLabel): string => (label as string).toString().replace(/Model [0-9]+/g, 'Models'),
+        priority: 0,
```

Comment:
Why priority 0? I don't think this should be shown 1st even above the standard description.